packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -49,6 +49,7 @@ void main() {
   Directory projectDir;
   FlutterVersion mockFlutterVersion;
   LoggingProcessManager loggingProcessManager;
+  BufferLogger logger = BufferLogger.test();
```

Comment:
Sorry, I also meant put the `logger = BufferLogger.test()` in setUp, match the pattern of `LoggingProcessManager`.

Reply:
Yeah I did that and forgot the remove the other constructor call. Fixed in a new commit.